hugegraph-server/hugegraph-dist/docker/README.md

```diff
@@ -35,7 +35,7 @@ If you want to customize the preloaded data, please mount the groovy scripts (no
 
 1. Using docker run
 
-    Use `docker run -itd --name=graph -p 8080:8080 -e PRELOAD=true -v /path/to/yourScript:/hugegraph/scripts/example.groovy hugegraph/hugegraph`
+    Use `docker run -itd --name=graph -p 8080:8080 -e PRELOAD=true -v /absolute/path/to/your/script:/hugegraph-server/scripts/example.groovy hugegraph/hugegraph`
```

Comment:
seems a little redundant/long for `absolute` & `your/script`? (better keep the path simple to read)